# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1292520241 28800
#      Thu Dec 16 09:24:01 2010 -0800
# Node ID a41acdfa97dcd0124ec00c4dfafb06b09b4b3868
# Parent  566249cf97fb12c3d877e09282c8c1bf7e726989
mk_undocumented_list: Update list of exception functions not requiring documentation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,11 +1,16 @@
+2010-12-16  Rik  <octave@nomad.inbox5.com>
+
+	* interpreter/doccheck/mk_undocumented_list: Update list of exception
+	functions not requiring documentation.
+
 2010-12-09  Doug Stewart <doug.dastew@gmail.com>
 
-        * interpreter/plot.txi: changed origial to origin in Quiver section
+	* interpreter/plot.txi: changed origial to origin in Quiver section
 
 2010-12-09  Kai Habel  <kai.habel@gmx.de>
 
 	* interpreter/plot.txi: Add pie3 function.
 
 2010-11-26  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/install.txi (Compiling Octave with 64-bit
diff --git a/doc/interpreter/doccheck/mk_undocumented_list b/doc/interpreter/doccheck/mk_undocumented_list
--- a/doc/interpreter/doccheck/mk_undocumented_list
+++ b/doc/interpreter/doccheck/mk_undocumented_list
@@ -74,27 +74,42 @@ besselh
 besseli
 besselk
 bessely
 build_bc_overloads_expected
 chdir
 comma
 debug
 exit
+F_DUPFD
+F_GETFD
+F_GETFL
 fntests
+F_SETFD
+F_SETFL
 gammaln
 geometryimages
 i
 inf
 interpimages
 inverse
 j
 J
 lower
 nan
+O_APPEND
+O_ASYNC
+O_CREAT
+O_EXCL
+O_NONBLOCK
+O_RDONLY
+O_RDWR
+O_SYNC
+O_TRUNC
+O_WRONLY
 paren
 plotimages
 SEEK_CUR
 SEEK_END
 semicolon
 setenv
 sparseimages
 tbcover
@@ -105,27 +120,31 @@ test_contin
 test_diag_perm
 test_error
 test_for
 test_func
 test_global
 test_if
 test_io
 test_null_assign
+test_parser
 test_prefer
 test_range
 test_recursion
 test_return
 test_slice
 test_sparse
 test_string
 test_struct
 test_switch
 test_system
 test_transpose
 test_try
 test_unwind
 test_while
-test_parser
 toc
 triu
 unimplemented
 upper
+ylabel
+ylim
+zlabel
+zlim
